labcontrol/db/support_files/db_patch.sql

```diff
@@ -477,15 +477,15 @@ CREATE INDEX idx_16s_library_prep_composition_1 ON labcontrol.library_prep_16s_c
 CREATE TABLE labcontrol.library_prep_shotgun_process (
 	library_prep_shotgun_process_id bigserial  NOT NULL,
 	process_id           integer  NOT NULL,
```

Comment:
I recall once in labcontrol they used `bigint` instead of `integer`. Is there a consideration for this change?

Reply:
Thanks for asking! I believe the standard integer in PostgreSQL 9 is faster than bigint, and still four bytes long - that would make for over 2 billion process IDs, even if we ignore signage. We're probably pretty good in this case.